bin/dash-network

```diff
@@ -44,7 +44,7 @@ fi
 docker run -ti --rm \
            --device /dev/net/tun \
            --cap-add=NET_ADMIN \
-           -v "$PWD:/networks" \
+           -v "$PWD/networks:/networks" \
```

Comment:
Actually, you suppose to run dash-network from the directory with configs. I guess it's described in README